website/app/styles/pages/application/stage.scss

```diff
@@ -36,4 +40,8 @@
       auto;
     margin: 0 auto 0 0;
   }
+
+  &.doc-page-stage--tabs {
+    --doc-page-offset-height: calc(var(--doc-page-header-height) + var(--doc-page-tabs-height));
+  }
```

Comment:
suggestion: my preference for class modifiers is to live as siblings. I don't think we need specificity here, but I appreciate that un-nesting it means we rely on definition order

Reply:
Implemented your suggestion! Let me know if it looks okay.

Reply:
Yes, all good from me